src/robot/model/keyword.py

```diff
@@ -29,43 +34,46 @@ class Keyword(BodyItem):
     repr_args = ('name', 'args', 'assign')
     __slots__ = ['_name', 'args', 'assign', 'type']
 
-    def __init__(self, name='', args=(), assign=(), type=BodyItem.KEYWORD, parent=None):
+    def __init__(self, name: str = '', args: Sequence[str] = (),
+                 assign: Sequence[str] = (), type: str = BodyItem.KEYWORD,
+                 parent: 'TestSuite|None' = None):
```

Comment:
`Keyword` parent can be at least `TestSuite`, `TestCase`, and `Keyword`, but I believe it could also be `For`, `If`, etc. Probably the typing should be just `BodyItem|None` here.

Reply:
Good point. Could also have `ModelObject`, but that's too broad.